composer.json

```diff
@@ -12,6 +12,7 @@
         "symfony/finder": "^7.3|^8.0"
     },
     "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
please revert this

Reply:
fa90aee5b1e2717678ad16969e16f3a59a609905